conf/app.routes

```diff
@@ -12,4 +12,5 @@ GET        /account/sign-out-survey            uk.gov.hmrc.economiccrimelevyacco
 GET        /account/sign-out                   uk.gov.hmrc.economiccrimelevyaccount.controllers.SignOutController.signOutNoSurvey()
 GET        /account/signed-out                 uk.gov.hmrc.economiccrimelevyaccount.controllers.SignOutController.signedOut()
 
+GET        /account/view-returns               uk.gov.hmrc.economiccrimelevyaccount.controllers.ViewYourReturnsController.onPageLoad()
```

Comment:
An integration test is missing for this route, i.e. ViewYourReturnsISpec?

Reply:
Done 😄